orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/testframework/steps/GeneralTestStep.java

```diff
@@ -55,11 +55,6 @@ public void thereIsAField(String fieldName) {
         this.state.addField(fieldName);
     }
 
-    @Given("the maximum string length is (\\d+)$")
```

Comment:
Why are we removing this step?

Reply:
isn't hooked up properly, and isn't used.

only currently used to 'set' the max length to what it already is.

Reply:
If it's not working properly then I agree with removing. If it works, then I vote we keep it to provide testers with a bigger toolkit to write tests without having to invoke developer help.

Resolve this if it is (which I think it is) the former.